### PR TITLE
Csv.fill

### PR DESCRIPTION
**Pull request by boxxxie**

### CODE REVIEW

**Comment by boxxxie** [comment]
@olafura i changed the namespace to Csv

File: lib/kitchen_sink/csv.ex [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+  ## Example
+
+       iex> KitchenSink.Csv.fill([1,"","",2,"",3,""], "")
+       [1,1,1,2,2,3,3]
+  """
+  def fill(sparse_list, empty_matcher) do
+    sparse_list_length = Enum.count(sparse_list)
+
+    {values, indecies} =
```
Comment (kenips) [inline lib/kitchen_sink/csv.ex]: indices?

File: test/kitchen_sink/csv_test.exs [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  alias KitchenSink.Csv, as: C
+
+  doctest KitchenSink.Csv
+
+  test "Csv.fill" do
+    expected = [1, 1, 1, 1, 1, 2, 2, 2, 2, 3, 3, 3, 3]
+    input = [1, "", "", "", "", 2, "", "", "", 3, "", "", ""]
+
+    assert expected == C.fill(input, "")
+  end
```
Comment (kenips) [inline test/kitchen_sink/csv_test.exs:14]: Not testing anything more than doctest. Should remove.

**Review by kenips — CHANGES_REQUESTED** [review]

File: lib/kitchen_sink/csv.ex (hunk 1 above)
Comment (boxxxie) [reply lib/kitchen_sink/csv.ex]: fixed

File: test/kitchen_sink/csv_test.exs (hunk 2 above)
Comment (boxxxie) [reply test/kitchen_sink/csv_test.exs:14]: new test

**Review by kenips — APPROVED** [review]

File: lib/kitchen_sink/csv.ex [hunk 3]
```diff
@@ -0,0 +1,33 @@
+defmodule KitchenSink.Csv do
```
Comment (olafura) [inline lib/kitchen_sink/csv.ex]: It should be all uppercase

File: test/kitchen_sink/csv_test.exs [hunk 4]
```diff
@@ -0,0 +1,15 @@
+defmodule KitchenSink.CsvTest do
```
Comment (olafura) [inline test/kitchen_sink/csv_test.exs]: ditto

**Review by olafura — CHANGES_REQUESTED** [review]

**Comment by boxxxie** [comment]
@olafura

**Comment by boxxxie** [comment]
i did the initializim

**Review by olafura — APPROVED** [review]

**Pull request merged**